Timing breakdown (#588)
elastic/apm-agent-java: docs/metrics.asciidoc

in bytes, allocated in heap memory.
--

[float]
[[metrics-application]]
=== Application Metrics

*`transaction.duration`*::
+
--
type: simple timer

> This timer tracks the transaction durations and allows to create graphs displaying a weighted average.

Reviewer: ```suggestion
This timer tracks the duration of transactions and allows for the creation of graphs displaying a weighted average.
```

Fields:

* `sum`: The sum of all transaction durations in ms since the last report (the delta)
* `count`: The count of all transactions since the last report (the delta)

You can filter and group by these dimensions:

* `transaction.name`: The name of the transaction
* `transaction.type`: The type of the transaction, for example `request`

--

*`transaction.breakdown.count`*::
+
--
type: long

format: count (delta)

> Tracks for how many transactions breakdown metrics (`span.self_time`) have been created.

Reviewer: This sentence is also confusing for me. What about:
```suggestion
The number of transactions for which breakdown metrics (`span.self_time`) have been created.
```
